travis-test.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+set -eu
+
+# Check that the servers build
+gb build github.com/matrix-org/dendrite/roomserver/roomserver
+gb build github.com/matrix-org/dendrite/roomserver/roomserver-integration-tests
+
+# Run the pre commit hooks
+./hooks/pre-commit
+
+# Run the integration tests
+${DEBUG:=""} bin/roomserver-integration-tests
```

Comment:
I used it in the script I cargoculted this line from to optional run the program under gdb or strace. Since I don't actually need to do that here I've removed it 8448117